regression/regression_var.sh

```diff
@@ -108,14 +108,14 @@ case $machine in
       export queue="${queue:-batch}"
       export group="${group:-global}"
       if [[ "$cmaketest" = "false" ]]; then
-	  export basedir="/work/noaa/da/$LOGNAME/gsi"
+	  export basedir="/work/noaa/epic-ps/$LOGNAME/gsi"
       fi
-      export ptmp="${ptmp:-/work/noaa/stmp/$LOGNAME/$ptmpName}"
+      export ptmp="${ptmp:-/work/noaa/epic-ps/$LOGNAME/ptmp}"
```

Comment:
This should revert to `/work/noaa/stmp/$LOGNAME/$ptmpName`

Reply:
changing back to /work/noaa/stmp/$LOGNAME/$ptmpName